common/src/main/java/dev/latvian/kubejs/server/TagEventJS.java

```diff
@@ -26,6 +27,10 @@
 /**
  * @author LatvianModder
  */
+@KubeJSEvent(
+		server = { "<ItemTypeName>.tags" },
```

Comment:
Not 100% on this, `ItemTypeName` isn't really too expressive

Reply:
Fixed in latest commit.